docs/src/components/NotFoundHero.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Section from 'docs/src/layouts/Section';
+import SectionHeadline from 'docs/src/components/typography/SectionHeadline';
+import SearchOffRoundedIcon from '@mui/icons-material/SearchOffRounded';
+
+function NotFoundIllustration() {
+  return (
+    <Box
+      sx={(theme) => ({
+        mx: 'auto',
+        mb: 4,
+        height: 150,
+        width: 200,
+        borderRadius: 1,
+        border: '1px solid',
+        borderColor: 'divider',
+        overflow: 'clip',
+        boxShadow: '0 4px 8px rgba(0, 0, 0, 0.05)',
```

Comment:
I think we could bump the box-shadows a bit, the borders look a bit muddy. Give it a try:
```suggestion
        boxShadow: '0px 2px 8px -2px rgba(21, 21, 21, 0.08), 0px 6px 12px -2px rgba(21, 21, 21, 0.08),
```